apps/test/unit/templates/progress/ProgressLessonTeacherInfoTest.js

```diff
@@ -4,6 +4,31 @@ import Immutable from 'immutable';
 import {shallow} from 'enzyme';
```

Comment:
While you're here, mind replacing deprecatedChai with reconfiguredChai?

Reply:
👍